Adafruit_AM2320.cpp

```diff
@@ -84,9 +84,15 @@ bool Adafruit_AM2320::begin() {
 /**************************************************************************/
 float Adafruit_AM2320::readTemperature() {
   uint16_t t = readRegister16(AM2320_REG_TEMP_H);
+  float ft;
   if (t == 0xFFFF) return NAN;
-
-  float ft = (int16_t)t;
+  // check sign bit - the temperature mSB is signd , bit 0-15 are mignitude
+  if(t >= 32768){
```

Comment:
how about `if (t & 0x8000)` to make it clear it's a bit check?

Reply:
sounds good - will implement